content/desktop/release-notes.md

```diff
@@ -57,6 +57,11 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 - Fixed a bug where proxy settings defined in `admin-settings.json` were not applied correctly on startup.
 - Fixed a bug where the **Manage Synchronized file shares with Compose** toggle did not correctly reflect the value with the feature.
 - Fixed a bug where a bind mounted file modified on host is not updated after the container restarts, when gRPC FUSE file sharing is used on macOS and on Windows with Hyper-V. Fixes [docker/for-mac#7274](https://github.com/docker/for-mac/issues/7274), [docker/for-win#14060](https://github.com/docker/for-win/issues/14060).
+- Build UI:
```

Comment:
```suggestion
- Builds view:
```

Reply:
Guess we should change previous release notes as well?: https://github.com/docker/docs/blob/79bb1b9421a80583d1048c0e71568fc455cac74c/content/desktop/release-notes.md?plain=1#L147